### PR TITLE
Add more analytics

### PR DESCRIPTION
**Pull request by qdequele**

### CODE REVIEW

**Comment by Kerollmops** [comment]
Have you tested that? Are these new values correctly sent and interpreted by Amplitude?

**Comment by qdequele** [comment]
Yes! Two screenshots of my tests.

Capture d’écran 2020-06-02 à 14 55 42
Capture d’écran 2020-06-02 à 14 55 02

**Comment by Kerollmops** [comment]
IIRC the `indexes.movies.fields` are there because this is an old screenshot, right?

**Comment by qdequele** [comment]
yes :s

File: README.md [hunk 1]
```diff
@@ -176,8 +176,8 @@ Hey! We're glad you're thinking about contributing to MeiliSearch! If you think
 
 ### Analytic Events
 
-Once a day, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
-Only information about the platform on which the server runs is stored. No other information is being sent.<br/>
+Every hours, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
+To see what information we're retrieving, please see the complete list on the dedicated [issue](https://github.com/meilisearch/MeiliSearch/issues/720).<br/>
```
Comment (Kerollmops) [inline README.md:180]:
```suggestion
To see what information we're retrieving, please see the complete list [on the dedicated issue](https://github.com/meilisearch/MeiliSearch/issues/720).<br/>
```

File: README.md [hunk 2]
```diff
@@ -176,8 +176,8 @@ Hey! We're glad you're thinking about contributing to MeiliSearch! If you think
 
 ### Analytic Events
 
-Once a day, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
-Only information about the platform on which the server runs is stored. No other information is being sent.<br/>
+Every hours, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
```
Comment (Kerollmops) [inline README.md:179]:
```suggestion
Every hour, events are being sent to our Amplitude instance so we can know how many people are using MeiliSearch.<br/>
```

**Pull request merged**